docs/pages/contributing/documentation/how-to-contribute.mdx

```diff
@@ -28,7 +28,7 @@ $ yarn git-update
 Next, navigate to the directory under `content` that corresponds to the latest version of Teleport.
 
 ```code
-$ cd content/(=version=)
+$ cd content/(=teleport.major_version=).x
```

Comment:
What's the reasoning behind this change?

Reply:
Well the change to the var makes sense because the `content` subdirectories have never included minor releases. The `.x` corresponds to a change made in the docs repo that changes the format of these subdirs from `##.0` to `##.x`.